xonsh/tools.py

```diff
@@ -285,6 +285,12 @@ class redirect_stderr(_RedirectStream):
     _stream = "stderr"
 
 
+def executables_in(path):
```

Comment:
Would you mind adding a test for this function please? It could be pretty simple.